src/client.ts

```diff
@@ -5288,10 +5288,6 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         ];
 
         if (this.supportsExperimentalThreads()) {
-            if (!timelineSet.canContain(event)) {
-                return undefined;
-            }
```

Comment:
Instead of returning `undefined`, we now just ignore the event and don't add it to the given `timelineSet`